service/controller/v22/resource/s3bucket/delete.go

```diff
@@ -23,62 +35,92 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 		return microerror.Mask(err)
 	}
 
-	for _, bucketInput := range bucketsInput {
-		if bucketInput.Name == "" {
-			continue
-		}
+	g, ctx := errgroup.WithContext(ctx)
```

Comment:
We delete different buckets in parallel now. This might not be too big of a deal in case the buckets vastly differ in the amount of objects they contain.